unified-codes/apps/web/src/components/app/Footer.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { BrowserRouter } from 'react-router-dom';
 import { ThemeProvider } from '@material-ui/core/styles';
 
 import { Footer } from './Footer';
-import theme from '../../muiTheme';
+import theme from '../../theme';
```

Comment:
Looks like it now lives at ../../styles/theme? I modified it in PR #215 just because the TS error was bothering me (I hope I got the right one)